src/util-base64.c

```diff
@@ -318,8 +319,8 @@ static int B64TestVectorsRFC2045(void)
     const char *src9 = "Zm$9vYm.Fy";
     const char *fin_str9 = "foobar";
 
-    const char *src10 = "Y21Wd2IzSjBaVzFoYVd4bWNtRjFaRUJoZEc4dVoyOTJMbUYxOmpqcHh4b3Rhb2w%5d";
-    const char *fin_str10 = "cmVwb3J0ZW1haWxmcmF1ZEBhdG8uZ292LmF1:jjpxxotaol9t";
```

Comment:
What is this test change about Shivani ?

Reply:
No idea. I don't even see it in this PR. Not sure how you reached here to comment on it 🤔 
It is labelled "Outdated" for me..

Reply:
I saw commit by commit

Reply:
oh. thanks.
So, this change was to make a string  such that after skipping all invalid chars, it is a perfectly divisible by 4 string which can be decoded to ascii chars that i can write in the test.
Earlier the len was 66 with an invalid char making it 65 even if that invalid char was skipped so I got rid of one char to be able to have a perfectly decoded string 😅